### PR TITLE
Docs: mention added fields in 3.3 with a note

### PR DESCRIPTION
**Pull request by kares**
The docs for [3.3](https://github.com/logstash-plugins/logstash-filter-useragent/pull/68) ended up not mentioning these added fields in the (ECS) table.

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -40,20 +40,22 @@ output.
...
+|[os_name]     |[user_agent][os][name]                              |Detected operating-system name |
+|[os_version]* |[user_agent][os][version]                           |Detected OS version |Only available in ECS mode
+|[os_major]    |[@metadata][filter][user_agent][os][version][major] |OS major version |Only as meta-data in ECS mode
+|[os_minor]    |[@metadata][filter][user_agent][os][version][minor] |OS minor version |Only as meta-data in ECS mode
+|[os_patch]    |[@metadata][filter][user_agent][os][version][patch] |OS patch version |Only as meta-data in ECS mode
+|[os_full]     |[user_agent][os][full]                              |Full operating-system name |
+|[device]      |[user_agent][device][name]                          |Device name |
 |=======================================================================
 
+NOTE: `[version]` and `[os_version]` fields were added in Logstash **7.14** and are not available by default in prior versions.
```
Comment (kares) [inline docs/index.asciidoc:57]: maybe there's a better way esp. considering we might add other fields later (in a next minor/major release).
first thought of adding a footnote but that did not work as expected.

Comment (karenzone) [inline docs/index.asciidoc:57]:
```suggestion
NOTE: `[version]` and `[os_version]` fields were added in Logstash **7.14** and are not available by default in earlier versions.
```

Comment (karenzone) [reply docs/index.asciidoc:57]: "Earlier" and "later" are generally accepted as the best terms to use for versioning info.

File: docs/index.asciidoc [hunk 2]
```diff
@@ -69,11 +71,11 @@ produces the following fields:
 -----
     {
         "name"=>"Firefox",
-        "version"=>"45.0",
+        "version"=>"45.0", # since plugin version 3.3.0
```
Comment (karenzone) [inline docs/index.asciidoc:74]: Nice! I like the way you included the new fields in the example, while making the version restrictions obvious.  :star:

**Review by karenzone — APPROVED** [review]
I left a minor comment inline for your consideration. 

Otherwise, builds cleanly and renders as I expected. LGTM 🚀 

Asciidoctor doesn't always render as expected, and so I mocked up a preview for you here so that you can see what it looks like: https://logstash-docs_1249.docs-preview.app.elstc.co/guide/en/logstash/master/plugins-filters-useragent.html

I think it looks good and clearly communicates the new info. Nice job!

File: docs/index.asciidoc [hunk 3]
```diff
@@ -40,20 +40,22 @@ output.
 |=======================================================================
 |ECS disabled |ECS v1 |Description |Notes
```
Comment (karenzone) [inline docs/index.asciidoc:41]:
```suggestion
|ECS disabled |ECS v1, v8 |Description |Notes
```

Comment (karenzone) [reply docs/index.asciidoc:41]: Should we add v8 here to be consistent with versioning in the `ecs_compatibility` section?

**Pull request merged**